combinations/hash.tsv

```diff
@@ -403,3 +403,4 @@ bedtools=2.31.0,samtools=1.17,coreutils=9.3,gawk=5.1.0
 bioconductor-tximeta=1.16.0,bioconductor-rhdf5=2.42.0
 bismark=0.24.0,seqkit=2.4.0,pigz=2.6,csvtk=0.25.0
 bbmap=39.01,samtools=1.17,seqkit=2.4.0,pigz=2.6,csvtk=0.25.0
+r-base=4.1.3,r-argparse=2.1.3,r-data.table=1.14.2,r-circlize=0.4.15,r-randomcolor=1.1.0.1,bioconductor-complexheatmap=2.10.0 
```

Comment:
you don't need to specify r-base, this gets pulled in automatically

Reply:
good to know! thank you!